### DISCUSSION TITLE
Playback, Change OBC sync by default.

### DISCUSSION THREAD

**Discussion by TFL999** [f1multiviewer/issue-tracker#205] [Feature requests]
When I watch the playback of a race, I regularly change the OBC's that I have open on my second screen ( to the one's with the most interesting battle.)
But when you change the OBC stream in the window by right-clicking by default it will start from session begin, also halfway a race.

It would be nice if the OBC would jump to the sync position of the rest of the streams that are open, making it easier to change OBC streams.

**Comment by NeuroDawg** [comment]
I would second this request. 

I like to open/close different driver cameras throughout the race replay (since I rarely watch live on the US west coast), and would love for them to open at the point in the race the other windows are at. This would be killer!

**Comment by S0und** [comment]
Also auto sync on playback events. If I pause the VOD, send a `s` sync and pause every other window.

**Comment by jeppeolesen** [comment]
This should be fixed in one of the many updates since this request was filed. If it remains to be an issue, pease file a bug report under issues: https://github.com/f1multiviewer/issue-tracker/issues
